doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,788 @@
+====
+
+This is a very *significant* "minor" release of the pbench-agent code base,
+primarily to deliver the new "Tool Meister" sub-system.  It also delivers:
+
+ * Tool registration kept local to the host where registration happens
+ * Support of Prometheus and PCP tool data collection
+ * Removal of gratuitous software installation, only checks for requirements
+   * True for both tools and benchmark script requirements
+   * Change to check command versions instead of RPM versions for `pbench-fio`, `pbench-linpack`, and `pbench-uperf`
+ * The `pbench-linpack` benchmark script now provides result graphs, JSON data files, and supports execution on one or more local / remote hosts
+ * Required use of `--user` with `pbench-move-results`/`pbench-copy-results`
+ * Latent support for new HTTP PUT method of posting tar balls
+ * Removal of the dependency on the SCL (Software Collections Library)
+ * Support for `pbench-trafficgen` benchmark script dropped entirely
+ * Deprecation announcements for unused benchmark convenience scripts:
+   * `pbench-run-benchmark`, `pbench-cyclictest`, `pbench-dbench`, `pbench-iozone`, `pbench-migrate`, and `pbench-netperf`
+ * Many, many, bug fixes and behavioral improvements
+
+You can review the [**Full ChangeLog**](https://github.com/distributed-system-analysis/pbench/compare/b0.69-bp...v0.71.0-alpha.0) on GitHub (all 550+ commits, tags `b0.69-bp` to `v0.71.0-alpha.0`), or read a summary with relevant details below.
+
+We did not bump the "major" release version number with these changes because we still don't consider all the necessary functionality in place for such a major version bump.
+
+Note that work on the `v0.71` release started in earnest with the `v0.69.3-agent` release (tagged as `b0.69-bp`).  A number of bug fixes and behaviors from the `v0.71` work have already been back-ported and delivered in the various `v0.69.*` releases since then.  These release notes will highlight only the behavioral changes that have not been back-ported previously.
+
+
+## The New "Tool Meister" Sub-System
+
+The "Tool Meister" sub-system (introduced by PR #1248) is the major piece of functionality delivered with the release of `v0.71` of the pbench-agent.
+
+This is a significant change, where the pbench-agent first orchestrates the instantiation of a "Tool Meister" process on all hosts registered with tools, using a Redis instance to coordinate their operation, and the new "Tool Data Sink" process handles the collection of data into the pbench run directory hierarchy.  This effectively eliminates all remote SSH operations for individual tools except one per host to orchestrate the creation of the Tool Meister instance.
+
+One Tool Meister instance is created per registered host, and then a single Tool Data Sink instance is created on the host where the benchmark script is run.  The Tool Data Sink is responsible for collecting and storing locally all data send to it created by the Tool Meister instances.
+
```

Comment:
The downside of your de-line-breaking is that it makes comments less specific.

s/all data send/all data sent/ ??? But I also think maybe the "created by" phrase doesn't really fit.